### DISCUSSION TITLE
tls support for mqtt

### DISCUSSION THREAD

**Discussion by GMagician** [0x2142/frigate-notify#159] [General]
Is there any way possible to connect to mqtt on port 8883? I can't see any certificate in config.

Thanks

**Comment by 0x2142** [comment]
Hi there,
  At the moment, TLS for MQTT isn't supported - but I'm happy to look into adding this in 👍

**Reply by GMagician** [reply]
Wow, great!!
I also see I can't connect to my frigate because of https. I'm using a self ca certificate and mDNS. Since nMDSN uses .local, I need to put it in config.ymlm but in certificate CN field I must use name without .local. This ends up in a authorization failure.
